RealmSwift/SwiftUI.swift

```diff
@@ -1206,7 +1206,12 @@ public extension BoundCollection where Value == List<Element> {
     /// :nodoc:
     func move(fromOffsets offsets: IndexSet, toOffset destination: Int) {
         safeWrite(self.wrappedValue) { list in
-            list.move(fromOffsets: offsets, toOffset: destination)
+            var d = destination
+            if let firstOffset = offsets.first,
+               destination > firstOffset {
+                d = destination - 1
+            }
+            list.move(fromOffsets: offsets, toOffset: d)
```

Comment:
`list.move(fromOffsets:toOffset:)` is our function and not something built-in, so if it's doing the wrong thing then it should be fixed.

Reply:
I moved the fix to list.move() @tgoyne